modules/dnn/test/test_darknet_importer.cpp

```diff
@@ -487,7 +487,8 @@ TEST_P(Test_Darknet_nets, YOLOv3)
 #if defined(INF_ENGINE_RELEASE)
     if (backend == DNN_BACKEND_INFERENCE_ENGINE_NN_BUILDER_2019)
     {
-        if (INF_ENGINE_VER_MAJOR_LE(2018050000) && target == DNN_TARGET_OPENCL)
+        if ((INF_ENGINE_VER_MAJOR_LE(2018050000) || INF_ENGINE_VER_MAJOR_GE(2020020000)) &&
+            target == DNN_TARGET_OPENCL)
             applyTestTag(CV_TEST_TAG_DNN_SKIP_IE_OPENCL, CV_TEST_TAG_DNN_SKIP_IE_VERSION);
         else if (INF_ENGINE_VER_MAJOR_EQ(2019020000))
```

Comment:
> INF_ENGINE_VER_MAJOR_GE(2020020000)
> else if (INF_ENGINE_VER_MAJOR_EQ(2019020000))

This code path should be updated too.

Also there are failed tests with OV 2019R3 and 2020.1

Reply:
Seems to me we can skip `DNN_TARGET_OPENCL` for all the versions and `DNN_TARGET_OPENCL_FP16` for <= 2020.1

Reply:
@dkurt Please check code compilation.

Reply:
Sorry, fixed that!